.github/actions/update-omnistrate-plan/action.yaml

```diff
@@ -33,6 +37,12 @@ runs:
 
     - name: Update plan using Omnistrate CTL
       shell: bash
+      working-directory: ${{ inputs.working-directory }}
       run: |
         echo "Updating plan with file: ${{ inputs.file }}"
-        omctl build -f "${{ github.workspace }}/${{ inputs.file }}" -n "${{ inputs.service-name }}" --environment "${{ inputs.environment }}" --environment-type "${{ inputs.environment-type }}" --release-as-preferred
+        if [ "${{ inputs.working-directory }}" = "." ]; then
+          FILE_PATH="${{ github.workspace }}/${{ inputs.file }}"
+        else
+          FILE_PATH="${{ github.workspace }}/${{ inputs.working-directory }}/${{ inputs.file }}"
+        fi
+        omctl build -f "$FILE_PATH" -n "${{ inputs.service-name }}" --environment "${{ inputs.environment }}" --environment-type "${{ inputs.environment-type }}" --release-as-preferred
```

Comment:
We should cd to the working directory and run the rest of the script relative to that